tfmigrate apply fails with "Not Found" on push event after successful migration

### tfaction version

v2.0.2

### Overview

On a `push` event after merging a PR that touches a tfmigrate working directory, the `tfmigrate apply` job fails immediately after the migrator reports `state migrator apply success!`, with `##[error]Not Found - https://docs.github.com/rest`.

The tfmigrate apply itself completes (state and history are saved). The 404 is raised during the post-apply PR-comment step. The most likely root cause is that the `apply` action's `comment` option does not set `pr`, so `getPRNumber()` in `src/comment/exec.ts` falls back to `github.context.issue.number`, which is `0` on a `push` event. Calling `octokit.rest.issues.createComment({ issue_number: 0, ... })` then returns 404.

For comparison, `src/actions/create-follow-up-pr/index.ts` resolves the PR number from `env.all.CI_INFO_PR_NUMBER`. In the same failing run, the `create-follow-up-pr` action successfully posts to the merged PR (visible as `Posted comment to the original PR` later in the log), confirming that `CI_INFO_PR_NUMBER` is accessible on this event; the `apply` action's comment path instead uses the fallback shown above.

### How to reproduce

This is a code-path issue determined by `src/actions/apply/tfmigrate.ts` and `src/comment/exec.ts:getPRNumber()`; it does not depend on user configuration. Any workflow built on `tfaction-example` that triggers `tfmigrate apply` via a `push` event after merging a PR reproduces it.

Environment:

- Production failure observed on: GitHub Actions self-hosted runner, linux / arm64, Node.js 24 (tfaction's `runs.using: node24`), tfaction v2.0.2
- Local reproduction not attempted: the failure surface involves the GitHub Actions runtime's event context (`github.context.issue.number` on `push`), which is not directly reproducible in a standalone Node script. The reasoning relies on reading tfaction's source and the observed CI log.

tfaction-root.yaml — Use the canonical example as-is: https://github.com/suzuki-shunsuke/tfaction-example/blob/c7fca907d6ea32c70fa86df2a9544c4676ee2946/tfaction-root.yaml

tfaction.yaml — Any working-directory `tfaction.yaml` defining a tfmigrate target (e.g. `github/services/foo/tfaction.yaml`).

GitHub Actions Workflow — Use the canonical example with `suzuki-shunsuke/tfaction@v2.0.2`: https://github.com/suzuki-shunsuke/tfaction-example/blob/c7fca907d6ea32c70fa86df2a9544c4676ee2946/.github/workflows/apply.yaml

Excerpt from a failing job log:

```
##[group]tfmigrate apply
...
2026/06/11 06:01:37 [INFO] [migrator] state migrator apply success!
2026/06/11 06:01:37 [INFO] [runner] add a record to history: <migration file>
2026/06/11 06:01:37 [INFO] [runner] save history
2026/06/11 06:01:37 [INFO] [runner] history saved
##[endgroup]
##[error]Not Found - https://docs.github.com/rest
##[group]Run suzuki-shunsuke/tfaction@<v2.0.2 SHA>
  action: create-follow-up-pr
...
Posted comment to the original PR
```

### Expected behaviour

After tfmigrate apply succeeds, the action posts a `tfmigrate-apply` comment to the merged PR (resolved from `CI_INFO_PR_NUMBER`), and the job exits successfully.

### Actual behaviour

After tfmigrate apply succeeds, the action's post-apply comment call hits `octokit.rest.issues.createComment` with `issue_number: 0` (the push-event fallback from `github.context.issue.number`), receives a 404 from the GitHub API, and the job exits as failed even though the migration itself was applied and recorded.

### Note

- Investigated with Claude Code (AI-assisted).
- For reference, the minimum-impact fix would be to set `pr: env.all.CI_INFO_PR_NUMBER` on the comment option in `src/actions/apply/tfmigrate.ts`, matching the pattern used in `src/actions/create-follow-up-pr/index.ts`.

--- a/src/actions/apply/tfmigrate.ts
+++ b/src/actions/apply/tfmigrate.ts
@@ -75,6 +75,7 @@ export const main = async (
         comment: {
           token: githubToken,
           key: "tfmigrate-apply",
+          pr: env.all.CI_INFO_PR_NUMBER,
           vars: {
             tfaction_target: targetConfig.target,
           },

